### ISSUE TITLE
Unable to map the real IP of the service to the server

### ISSUE THREAD

**Issue by cyxnzb** [WisdomShell/codeshell-vscode#9]
I tried --host 127.0.0.1 but couldn't -- host my server's real IP.
Does it mean that I only need --host 127.0.0.1 to access this project through the real IP: port of my server?
<img width="605" alt="image" src="https://github.com/WisdomShell/codeshell-vscode/assets/51886383/163a1cf6-c4d0-4dcb-be58-e7df45236a02">

**Comment by ruixie** [comment]
Of course, you can bind to your actual IP using the `--host IP` option. If that doesn't work, you can attempt to listen on all IPs by using `--host 0.0.0.0`.

**Comment by cyxnzb** [comment]
> Of course, you can bind to your actual IP using the `--host IP` option. If that doesn't work, you can attempt to listen on all IPs by using `--host 0.0.0.0`.

好像绑定不了我的服务器IP，只能0.0.0.0

**Comment by niuzhi** [comment]
服务能正常启用，页面也可以访问到，但是在vscode插件里面总是提示超时，；链接不上codeshell，这是什么原因？
1.codeshell 服务
 
![image](https://github.com/WisdomShell/codeshell-vscode/assets/33443724/13772595-27ce-4081-b923-c6aaa9a36391)

2. 浏览器访问codeshell 服务正常
   
![image](https://github.com/WisdomShell/codeshell-vscode/assets/33443724/28cddbb4-7778-45f7-8987-038810e2c328)

3.vscode 插件配置
   
![image](https://github.com/WisdomShell/codeshell-vscode/assets/33443724/e51db03f-aeb4-4465-b0f9-9d47f329dfa7)

4.vscode 调用codeshell插件进行分析，发现链接不上codeshell 服务，什么原因？
   
![image](https://github.com/WisdomShell/codeshell-vscode/assets/33443724/fa037523-2879-4ce8-9ebd-198242b60eab)

**Comment by DongchenZhou** [comment]
I have also encountered this, which may be due to the CPU's excessively fast inference speed, causing the request to think that it is a time supermarket
